app/lib/frontend/handlers/publisher_client.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:client_data/publisher_api.dart' as p;
+import 'package:http/http.dart' as http;
+import 'package:api_builder/_client_utils.dart' as utils;
+
+export 'package:api_builder/_client_utils.dart' show RequestException;
+
+class PublisherApiClient {
+  final utils.Client _client;
+
+  PublisherApiClient(String baseUrl, {http.Client client})
```

Comment:
What's the benefit of using this over the generated `PublisherApiClient`?

Reply:
oh, my bad this was the hand written code I aimed to create with the generator... this should just be deleted :) hehe